tensorflow/core/kernels/split_lib_gpu.cu.cc

```diff
@@ -217,10 +221,10 @@ void SplitVOpGPULaunch<T, IntType>::Run(
     GpuLaunchConfig config =
         GetGpuLaunchConfig(total_rows * total_cols, gpu_device);
 
-    TF_CHECK_OK(CudaLaunchKernel(SplitVOpKernel_fixed<T>, config.block_count,
-                                 config.thread_per_block, 0,
-                                 gpu_device.stream(), input_ptr, total_rows,
-                                 total_cols, output_ptr_data));
+    TF_CHECK_OK(
+        GpuLaunchKernel(SplitVOpKernel_fixed<T>, dim3(config.block_count),
```

Comment:
Does ROCm's dim3 not implicitly construct from int?

Reply:
it does...the dim3 call does not need to be there...will remove it along with the extra parens